core/main.go

```diff
@@ -181,12 +168,22 @@ func main() {
 			logrus.WithField("module", "main").Fatalln(err)
 		}
 	}()
+
+	// init gRPC server.
+	server := grpc.New(rpc, ep, orch, cfg.AuthorizedPubKeys)
```

Comment:
had to reverse order of init of the gRPC server as it now requires the orchestrator as dep